go.mod

```diff
@@ -40,8 +40,8 @@ require (
 	github.com/opencontainers/selinux v1.6.0
 	github.com/openshift/api v0.0.0
 	github.com/openshift/client-go v0.0.0
-	github.com/operator-framework/go-appr v0.0.0-20180917210448-f2aef88446f2 // indirect
 	github.com/openshift/library-go v0.0.0-20200821154433-215f00df72cc
+	github.com/operator-framework/go-appr v0.0.0-20180917210448-f2aef88446f2 // indirect
```

Comment:
Please revert this file, not related, as its not changed

Reply:
Done.